swagger.yaml

```
@@ -1,11 +1,235 @@
 swagger: '2.0'
 info:
-    version: '1.0.0'
-    title: 'E-Commerce API'
-    description: 'API for managing brands, products, and user cart'
-host: 'localhost:3000'
+    title: Sunglasses.io
+    description: An API to support the Sunglasses-io platform
+    version: 1.0.0
+host: api.sunglasses.com
 schemes:
-    - 'http'
-basePath: '/api'
+    - https
+basePath: /v1
 produces:
-    - 'application/json'
+    - application/json
+tags:
+    - name: 'Store'
+      description: 'Accessing the sunglasses products'
+    - name: 'User'
+      description: 'Operations for the User'
+
+paths:
+    /store/brands:
+        get:
+            tags:
+                - 'Store'
+            summary: 'Returns Brands of sunglasses sold by store'
+            description: 'Returns a list of brands available'
+            responses:
+                200:
+                    description: Succesfull Request
+                    schema:
+                        type: array
+                        items:
+                            $ref: '#/definitions/Brand'
+    /store/brands/{Id}/products:
```

make sure to use lowercase for id.
```suggestion
    /store/brands/{id}/products:
```